src/Test/Integration/Data/Payment/Braintree/CreditCard.php

```diff
@@ -31,9 +31,11 @@ class CreditCard
     /**
      * Outsourced into a function for re-use
      *
+     * @param string|null $cartType
+     *
      * @return array
      */
-    public static function getAdditionalPayment(): array
+    public static function getAdditionalPayment($cartType = null): array
```

Comment:
can't be null, can it?

Reply:
yes, in case of "Not mapped payment method"

Reply:
kk